Fabric.Authorization.AccessControl/src/app/models/authMemberSearchRequest.model.ts

```diff
@@ -1,9 +1,7 @@
-import { AuthMemberSearchResult } from './authMemberSearchResult.model';
-
 export type SortDirection = 'asc' | 'ascending' | 'desc' | 'descending';
 export type SortKey = 'name' | 'role' | 'lastlogin' | 'subjectid';
 
-export interface AuthMemberSearchRequest {
+export interface IAuthMemberSearchRequest {
```

Comment:
changed all models to interfaces as they are much easier to initialize and work with

Reply:
There is a debate whether we should prefix them with `I` eg. IRole, as that is a c# standard not a ts standard. I can go either way

Reply:
I do not have a strong preference either way. If it makes it easier to read the code then I am fine with keeping it